src/MCPServer/lib/linkTaskManagerReplacementDicFromChoice.py

```diff
@@ -46,27 +46,33 @@ def __init__(self, jobChainLink, pk, unit):
         for i, dic in enumerate(dicts):
             self.choices.append((i, dic.description, dic.replacementdic))
 
-        # This is an attempt to find a choice in DashboardSettings for the
-        # chain link that the current MicroServiceChoiceReplacementDic is
-        # taking us. We're looking up the setting dict by the module name
-        # that the StandardTaskConfig wants to execute, e.g. upload-qubit_v0.0.
+        # There are MicroServiceChoiceReplacementDic links with no
+        # replacements (``self.choices`` has zero elements at this point). This
+        # is true for the following chain links:
         #
-        # Not implemented yet, but this could enable us to store more than a
-        # configuration set a specific chain link.
+        #   - ``Choose Config for Archivists Toolkit DIP Upload``
+        #   - ``Choose config for AtoM DIP upload``
+        #   - ``Choose Config for ArchivesSpace DIP Upload``
         #
-        # DashboardSettings does not belong to MCPServer. We currently have
-        # direct access to the database but that may not be always possible.
-        try:
-            mscl = MicroServiceChainLink.objects.get(id=jobChainLink.pk)
-            task_id = mscl.defaultnextchainlink.currenttask.tasktypepkreference
-            stc = StandardTaskConfig.objects.get(id=task_id)
-        except (MicroServiceChainLink.DoesNotExist, StandardTaskConfig.DoesNotExist, AttributeError):
-            pass
-        else:
-            args = DashboardSetting.objects.get_dict(stc.execute)
-            if args:
-                args = {'%{}%'.format(key): value for key, value in args.items()}
-                self.choices.append((len(self.choices), stc.execute, str(args)))
```

Comment:
I've moved this look up down to `_get_dashboard_setting_choice`, much cleaner that way!